#! /bin/sh /usr/share/dpatch/dpatch-run
## 13-add-QtReactor_export.h.dpatch
##
## DP: add missing header to libace-qtreactor-dev.

@DPATCH@
--- ACE_wrappers.orig/ace/QtReactor/ace_qt4reactor.mpc	2010-09-17 22:16:53.000000000 +0200
+++ ACE_wrappers/ace/QtReactor/ace_qt4reactor.mpc	2010-12-30 20:49:42.598080555 +0100
@@ -16,6 +16,10 @@
   MOC_Files {
   }
 
+  Header_Files {
+    ACE_QtReactor_export.h
+  }
+
   Source_Files {
     QtReactor_moc.cpp
     QtReactor.cpp
